### PR TITLE
thwo/handle-unkeyed-payload-v2

### PR DESCRIPTION
**Pull request by wogelius**
Fix bug where RI view would crash if payload is unkeyed

### CODE REVIEW

File: src/RequestInsurance/Models/RequestInsurance.php [hunk 1]
```diff
@@ -449,7 +436,22 @@ public function getHeadersWithMaskingApplied(): string
      */
     public function getPayloadWithMaskingApplied(): string
     {
-        return $this->getAttributeWithMaskingApplied('payload');
+        $payload = $this->getJsonDecodedPayload();
+
+        // Only process payload if it is an array
+        if ( ! is_array($payload)) {
+            return json_encode($payload, JSON_THROW_ON_ERROR);
```
Comment (nizacegodk) [inline src/RequestInsurance/Models/RequestInsurance.php:443]: Hvorfor json encoder du den her?

Comment (nizacegodk) [reply src/RequestInsurance/Models/RequestInsurance.php:443]: Caset, er vel RAW payload

Comment (wogelius) [reply src/RequestInsurance/Models/RequestInsurance.php]: Ja du har ret - fikset

**Review by nizacegodk — APPROVED** [review]
LGTM

**Pull request merged**